ocs_ci/ocs/platform_nodes.py

```diff
@@ -3243,9 +3244,28 @@ def stop_nodes(self, nodes, wait=True):
         self.ibmcloud_bm.stop_machines(machines)
         if wait:
             node_names = [n.name for n in nodes]
-            wait_for_nodes_status(
-                node_names, constants.NODE_NOT_READY, timeout=180, sleep=5
-            )
+            try:
+                wait_for_nodes_status(
+                    node_names, constants.NODE_NOT_READY, timeout=180, sleep=5
+                )
+            except CommandFailed as err:
+                # Consider the situation where the cluster is not accessible after nodes are down,
+                # example: when all the nodes are off
+                if "Unable to connect to the server" in str(err):
+                    logger.info(
+                        f"Unable to connect to the server to check the nodes NotReady status. "
+                        f"Trying alternate method to verify nodes {node_names} has stopped"
+                    )
+                    machines_not_off = (
+                        self.ibmcloud_bm.check_if_machines_are_off_from_sensor_data(
+                            machines
+                        )
+                    )
+                    assert not (
```

Comment:
I would rename the variable as active_machines to mitigate the double negative.

Reply:
Renamed the method to understand what the it returns.
Still, I would like to distinguish  between "ON" and "not OFF". The function is not recommended to identify the machines that are "ON".